### PR TITLE
created adaptives for team, streedcode sliders

### PR DESCRIPTION
**Pull request by FlSlaves**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

Lack of adaptives for mobile, tablet

## Summary of change

Added adaptives for tablet and mobile.

## Testing approach

--

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/features/MainPage/TeamSlider/TeamItemSlider/TeamItemSlider.styles.scss [hunk 1]
```diff
@@ -13,23 +17,59 @@
         line-height: f.pxToRem(23px);
         background-color: #FFFFFF;
         margin-left: 20px;
+        @media (min-width: 768px) and (max-width: 1024px) {
+            margin-top: 40px;
+                @include mut.sized(266px, 402px);
+                padding-bottom:20px;
+            }
+        @media screen and (max-width: 767px){
```
Comment (MaksBrat) [inline src/features/MainPage/TeamSlider/TeamItemSlider/TeamItemSlider.styles.scss:25]: use pxToRem()

**Review by MaksBrat — APPROVED** [review]

File: src/features/MainPage/StreetcodeSlider/StreetcodeSliderItem/StreetcodeSliderItem.styles.scss [hunk 2]
```diff
@@ -43,36 +64,76 @@
...
     }
 
     .streetcodeMainPageContainer {
         box-sizing: border-box;
         width: 100%;
         height: 100%;
-        @include mut.flexed(column, start, space-between, nowrap);
+         @include mut.flexed(column, start, space-around, nowrap);
+        display: flex;
+
```
Comment (MaksBrat) [inline src/features/MainPage/StreetcodeSlider/StreetcodeSliderItem/StreetcodeSliderItem.styles.scss:131]: remove blank line

**Comment by MaksBrat** [comment]
overrall, please check your code and fix indentations, remove blank likes and use pxToRem()

File: src/features/MainPage/StreetcodeSlider/StreetcodeSliderItem/StreetcodeSliderItem.styles.scss [hunk 3]
```diff
@@ -26,14 +33,28 @@
...
         @include mut.full-rounded(50px);
         @include mut.parent-sized();
         @include mut.flex-centered();
+        @media (min-width: 768px) and (max-width: 1024px) {
+                @include mut.sized(100%, 338px);
+                border-bottom: 0px !important; 
+            }
+        @media screen and (max-width: 767px) {
+                @include mut.sized(100%, 398px);
+                border-bottom: 0px !important;
```
Comment (NadiaKishchuk) [inline src/features/MainPage/StreetcodeSlider/StreetcodeSliderItem/StreetcodeSliderItem.styles.scss:56]: please, don't use important if it is not except the critical situation

File: src/features/MainPage/StreetcodeSlider/StreetcodeSliderItem/StreetcodeSliderItem.styles.scss [hunk 4]
```diff
@@ -43,36 +64,76 @@
...
         flex: 1 1 auto;
         position: relative;
+        @media screen and (max-width: 1024px) {
+                height: fit-content;    
+                .textContainer {
+                    height: 100%;
+                    width: 100%;
+                    display: flex;
+                    flex-direction: column;
+                    text-align: center;
```
Comment (NadiaKishchuk) [inline src/features/MainPage/StreetcodeSlider/StreetcodeSliderItem/StreetcodeSliderItem.styles.scss:77]: use our mixin flexed instead

File: src/features/MainPage/StreetcodeSlider/StreetcodeSliderItem/StreetcodeSliderItem.styles.scss [hunk 5]
```diff
@@ -43,36 +64,76 @@
...
+            width: 100%;
+            position:unset;
+            text-decoration: underline;
+      }
   }
 
     .streercodeTitle {
         @include mut.rem-padded($top: 18px, $right: 0px, $bottom: 0px, $left: 0px);
         @include mut.with-font(ft.$closer-text-font, $font-weight: 500, $font-size: 34px);
         color: #1D1F23;
```
Comment (NadiaKishchuk) [inline src/features/MainPage/StreetcodeSlider/StreetcodeSliderItem/StreetcodeSliderItem.styles.scss:120]: don't use the color here as a string, it is better to set it as scss variable and use it then.

File: src/features/MainPage/TeamSlider/TeamItemSlider/TeamItemSlider.styles.scss (hunk 1 above)
Comment (NadiaKishchuk) [inline src/features/MainPage/TeamSlider/TeamItemSlider/TeamItemSlider.styles.scss:25]: the same here, don't use color string in the styles and don't use px, use the pxToRem function

**Review by NadiaKishchuk — CHANGES_REQUESTED** [review]

**Review by MaksBrat — APPROVED** [review]

**Comment by KateYatsiuk** [comment]
Please, fill in the pull request details correctly. Why are the sections "Assignees", "Project", "Development" empty?

**Review by KateYatsiuk — APPROVED** [review]

**Review by Dobriyr — CHANGES_REQUESTED** [review]
image
The text should be centered vertically
Tablet version

---
image
unnecessary side indentation
Tablet version

---

image
It is worth adding a restriction to show no more than 3 slides
Tablet version

**Pull request closed without merging**